odoo/addons/base/tests/test_mail.py

```diff
@@ -837,3 +837,27 @@ def test_single_email_re(self):
                 res, exp,
                 'Seems single_email_re is broken with %s (expected %r, received %r)' % (src, exp, res)
             )
+
+
+class TestMailTools(BaseCase):
```

Comment:
It's fine for now, this has some potential to create strange side issues of previous code expecting a value and now we return a Falsy one, so master will allow for early testing ^^

Now that we have a separate commit, it will be easy to backport later in case we need it.